connmgr.go

```diff
@@ -155,7 +156,15 @@ func (cm *BasicConnMgr) getConnsToClose(ctx context.Context) []inet.Conn {
 
 	// Sort peers according to their value.
 	sort.Slice(candidates, func(i, j int) bool {
-		return candidates[i].value < candidates[j].value
+		left, right := candidates[i], candidates[j]
+		// temporary peers are preferred for pruning.
+		if left.temp {
+			return true
```

Comment:
We should look at the actual values if `right.temp` is true too.

Reply:
Makes sense.